src/main/java/com/speaker/service/util/StringParser.java

```diff
@@ -0,0 +1,31 @@
+package com.speaker.service.util;
+
+import java.util.Optional;
+
+import static java.util.Objects.nonNull;
+
+public class StringParser {
+    private static final Integer LIMIT_VALUE = 2;
+
+    public static Optional<Pair<String, String>> parseOptionalStringBySpace(String names) {
```

Comment:
do not see a need for this method

Reply:
Comments accepted, corrections made. Method has been deleted